dags/longitudinal.py

```diff
@@ -24,8 +24,14 @@
     execution_timeout=timedelta(hours=12),
     instance_count=40,
     release_label="emr-5.11.0",
-    env={"date": DS_WEEKLY, "bucket": "{{ task.__class__.private_output_bucket }}"},
-    uri="https://raw.githubusercontent.com/mozilla/telemetry-airflow/master/jobs/longitudinal_view.sh",
+    env=tbv_envvar(
+        "com.mozilla.telemetry.views.LongitudinalView",
+        {
+            "bucket": "{{ task.__class__.private_output_bucket }}",
+            "to": DS_WEEKLY
+        },
+        metastore_location="s3://telemetry-parquet/longitudinal"),
```

Comment:
instead of adding a parameter here, you should just use other to pass it:
`other={"METASTORE_LOCATION": "s3://telemetry-parquet/longitudinal"}`

Reply:
I thought it might be a good idea to include it explicitly, since we'll be making a similar change to all of the other jobs (at some point - it's on the docket for soonish)